test/app.ts

```diff
@@ -3,11 +3,13 @@ import fastify from 'fastify';
 import { CompleteMember } from '@graasp/sdk';
 
 import registerAppPlugins from '../src/app';
+import { resetDependencies } from '../src/di/utils';
 import { Actor } from '../src/services/member/entities/member';
 import { saveMember } from '../src/services/member/test/fixtures/members';
 import { DB_TEST_SCHEMA } from './constants';
 
 const build = async ({ member }: { member?: CompleteMember | null } = {}) => {
+  resetDependencies();
```

Comment:
Necessary only for test and dev? Maybe let's wrap it around some if condition on the environment.

Reply:
You woud like to wrap the usage of `resetDependencies` with a if else in the build function ? I'm wondering if it is really necessary because we are using `build` only in the tests ? but maybe I misunderstand your comment.